parm/config/gefs/config.base

```diff
@@ -203,13 +199,11 @@ case "${APP}" in
 
     if [[ "${APP}" =~ A$ ]]; then
         export DO_AERO="YES"
-        export AERO_ANL_RUN="both"
-        export AERO_FCST_RUN="gdas"
+        export DO_AERO_FCST="NO"
```

Comment:
This will change how it has been running.  Since, previously, `AERO_FCST_RUN="gdas"`, the aerosol forecasts were disabled for `gefs`.  Are you sure this should be enabled?

Also, I'm not seeing a `MEMBER`-based option in `config.efcs`.  Could you point me to that?

Reply:
Yes, `AERO_FCST_RUN` should just be `DO_AERO` for gefs. If it is not that way right now, it's in error.

For `efcs`, it is commented out right now, but there is a section at the very top for turning off components, which is basically the same as `(( MEMBER > 0 ))`. This is likely to be needed again to turn off aerosols in the perturbed members (still awaiting final word).

Reply:
Yeah it looks like it was a copy/paste error in https://github.com/NOAA-EMC/global-workflow/pull/2491.  I've turned it back on now.